### DISCUSSION TITLE
Nested mapping

### DISCUSSION THREAD

**Discussion by rudolfbruder** [Jeroen-G/Explorer#78] [Q&A]
Hi,

I wonder how to achieve the nested mapping. As I am reading the docs I see this in the nested mapping section

   [
        'id' => 'keyword',
        'title' => 'text',
        'published' => 'boolean',
        'created_at' => 'date',
        'author' => [
            'name' => 'Jeroen',
        ],
    ]

Is this what should be inserted in the model? Because if yes then author 'name' => is fixed Jeroen here, should it be 'text'? I am asking because I raised discussion few months back on the topic of how to get this working with nested relationship and this part is the trickiest for me.

I am interested in the way how it works when I want to index some model, how do I specify that author 'name' in the index should contain  the $post->author->name value.

Link to your page: https://github.com/Jeroen-G/Explorer/blob/master/docs/mapping.md

**Comment by Jeroen-G** [comment]
> Because if yes then author 'name' => is fixed Jeroen here, should it be 'text'?

I think you are right 🙈

**Reply by gcjbr** [reply]
Is there an answer to that?

Simply calling the relationship returns a null value

**Reply by Jeroen-G** [reply]
You were right.
